content/en/docs/concepts/security/service-accounts.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Service Accounts
+description: >
+  Learn about the Kubernetes ServiceAccount object.
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
+This page introduces the ServiceAccount object in Kubernetes, providing
+information about how service accounts work, use cases, limitations,
+alternatives, and links to resources for additional guidance.
+
+<!-- body -->
+
+## What are service accounts? {#what-are-service-accounts}
+
+A service account is a type of non-human account that, in Kubernetes, provides
+a distinct identity in a Kubernetes cluster. Application Pods, system
+components, and entities inside and outside the cluster can use a specific
+ServiceAccount's credentials to identify as that ServiceAccount. This identity
+is useful in various situations, including authenticating to the API server or
+implementing identity-based security policies.
```

Comment:
I'm happy with this - enough detail not to mislead, concise enough that the reader ought to be comfortable continuing with the rest of the page.